src/server/game/Movement/MovementGenerators/TargetedMovementGenerator.cpp

```diff
@@ -48,6 +48,9 @@ void TargetedMovementGeneratorMedium<T, D>::_setTargetLocation(T* owner, bool up
     {
         if (!i_offset)
         {
+            if (i_target->IsWithinDistInMap(owner, 1.0f))
```

Comment:
What do you mean by "bounding box"?

By the way, [_IsWithinDist](https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/game/Entities/Object/Object.cpp#L1124) takes into account the objectSize of both the moving unit and the target.